k8-util/helm/Makefile

```diff
@@ -23,8 +24,8 @@ dry-run-aws:
 	helm install --debug --dry-run goodly-guppy ./fluvio --set cloud=aws
 
 update_version:
```

Comment:
Much of this stuff in this Makefile is obsolete with inline chart mechanism.   Chart is now compiled into installer rather than as separate independent install.

Reply:
It's not documented so I had doubts about something like that, let me know what could be a proper fix

Reply:
Let me clean up to reflect current state of helm chart installation and add more notes.

Reply:
Thanks, let me know if I could help

Reply:
This PR should fix the test issue:  https://github.com/infinyon/fluvio/actions/runs/1455781206